qiskit_machine_learning/kernels/fidelity_quantum_kernel.py

```diff
@@ -215,7 +215,7 @@ def _get_kernel_entries(self, left_parameters: np.ndarray, right_parameters: np.
                 left_parameters,
                 right_parameters,
             )
-            kernel_entries = np.real(job.result().fidelities)
+            kernel_entries = job.result().fidelities
```

Comment:
I've removed this use of `np.real()`. Behind the scenes I think this was effectively just using `np.asanyarray`, as the fidelities should be a sequence of floats. These entries are later added to a np.ndarray with np.float64 dtype anyway.